src/app/clusters/ota-requestor/GenericOTARequestorDriver.cpp

```diff
@@ -48,6 +48,8 @@ using namespace app::Clusters::OtaSoftwareUpdateRequestor::Structs;
 constexpr uint32_t kDelayQueryUponCommissioningSec = 30; // Delay before sending the initial image query after commissioning
 constexpr uint32_t kImmediateStartDelaySec         = 1;  // Delay before sending a query in response to UrgentUpdateAvailable
 constexpr System::Clock::Seconds32 kDefaultDelayedActionTime = System::Clock::Seconds32(120);
+System::Clock::Seconds32 mDelayConfirmCurrentImageSec =
+    System::Clock::Seconds32(0); // Delay before sending notify update applied command
```

Comment:
Can we update the comment to match the variable as well? e.g., Delay before confirming current image (in seconds)

Reply:
Sure, done